### DISCUSSION TITLE
Windows Application name in the task manager

### DISCUSSION THREAD

**Discussion by sproket** [fvarrui/JavaPackager#405] [General]
Hi,

Great tool!

When I build my application with maven it runs fine but the task manager shows my application as OpenJDK Platform binary.  Is there a way to show it here as my exe name?

**Comment by fvarrui** [comment]
Hi @sproket!

If you use launch4j, its executable will run java.exe as another process, so your main EXE will not appear in Task Manager, but java.exe.

Try using `winConfig.exeCreationTool=winrun4j`, so JP will use another type of EXE, which loads jvm.dll in order to run java, instead of calling java.exe and create another process. This way all it's done by one process.

**Reply by sproket** [reply]
Thanks! This works great. Is there any way to specify additional parameters to winrun4j? I see it has settings to imbed the ini and jar inside and specify splash.

**Reply by fvarrui** [reply]
I'm afraid not, but if posible we can add these features. Please, could you provide an example or a link explaining the process? 
Thanks

**Comment by fvarrui** [comment]
> Great tool!

Thx! 😃

**Comment by sproket** [comment]
:)
